assignmentPR/conversionF1.c

```diff
@@ -0,0 +1,33 @@
+// CS499-Group01-SiteScribe
+// Pull Request Assignment
+// 4th Sept 2018
+// Owner: Colton Nunley
+// Contributers: Joseph Remy, Julian Bell, and Tanner Massahos
+
+char number[] = "123.456";
+int c, n, d;
+
+// Returns the part of a float before the decimal (like a floor function)
```

Comment:
These really are quite stubby

Reply:
Agreed, but lets give constructive feedback please.

Reply:
let's*

Reply:
I should have also mentioned the lack of punctuation in the first comment...